CompatBot/readme.md

```diff
@@ -37,7 +37,7 @@ Next we open the databases and run [migrations](https://docs.microsoft.com/en-us
 * BotDb is used to store all the settings and custom data for the bot.
 * ThumbsDb is used to store PSN metadata and game thumbnail links.
 
-When databases are ready, we immediately restore bot [runtime statistics](Databese/Providers/).
+When databases are ready, we immediately restore bot [runtime statistics](Database/Providers/).
```

Comment:
Hopefully this typo is only limited to the readme and not an actual directory. I can remove this change if I'm breaking something.